### ISSUE TITLE
Editor fails to export atlas of single 3kx2k image

### ISSUE THREAD

**Issue by badlogic** [EsotericSoftware/spine-editor#151]
Settings used 

![https://libgdx.badlogicgames.com/uploads/Screen%20Shot%202017-01-30%20at%2011.20.29%20AM-KbhTkg2o6c.png](https://libgdx.badlogicgames.com/uploads/Screen%20Shot%202017-01-30%20at%2011.20.29%20AM-KbhTkg2o6c.png)

Spine generates a `.atlas` file, but it is empty and not atlas page `.png` files are generated.

**Comment by NathanSweet** [comment]
Were we able to repro this? Sometimes a GPU reports a larger max texture size than it actually supports, then fails spectacularly.

**Comment by badlogic** [comment]
Yes, I did repro this locally. Image displays fine in the editor, doesn't get exported to atlas.

**Comment by NathanSweet** [comment]
Very old bug, haven't had any new reports. Spine's export fails gracefully with an error message when the GPU can't provide buffers large enough.
